engine/apps/alerts/models/alert_receive_channel.py

```diff
@@ -638,6 +642,21 @@ def insight_logs_metadata(self):
             result["team"] = "General"
         return result
 
+    @property
+    def alert_group_labels(self) -> IntegrationAlertGroupLabels:
+        """
+        Alert group labels configuration for the integration used by AlertReceiveChannelSerializer.
+        See AlertReceiveChannelAssociatedLabel.inherit for more details.
+        """
+        return {"inherit": {label.key_id: label.inherit for label in self.labels.all()}}
```

Comment:
Any reason not to just keep a list of the transmissible labels?

Reply:
No particular reason AFAICT, I agree a list could work better here